tests/net/socket/websocket/src/main.c

```diff
@@ -412,10 +430,16 @@ ZTEST(net_websocket, test_send_and_recv_empty_pong)
 
 	test_msg_len = 0;
 
-	ret = websocket_send_msg(POINTER_TO_INT(&ctx), NULL, test_msg_len, WEBSOCKET_OPCODE_PING,
+	fd = z_reserve_fd();
+	zassert_not_equal(fd, -1, "Failed to allocate FD");
+	z_finalize_fd(fd, &ctx, NULL);
```

Comment:
This snippet is repeated multiple times, perhaps we could create a small function for it?

Reply:
Added helper for FD allocation.